cpp/src/arrow/table.h

```diff
@@ -327,6 +327,13 @@ class ARROW_EXPORT Table {
   /// However, they may be equal even if they have different chunkings.
   bool Equals(const Table& other) const;
 
+  // Returns a new table of the same schema and contents but with all the
```

Comment:
Actually, can you please fix the comment sto be consistent with the rest of the file triple slashes  ("///")  and add a \brief?